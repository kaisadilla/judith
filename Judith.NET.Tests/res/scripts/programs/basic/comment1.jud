-- println("This should not be printed.")
-- println("This shouldn't either.")
println("The only line that has to be printed.")
-- println("If this prints, something is broken.")