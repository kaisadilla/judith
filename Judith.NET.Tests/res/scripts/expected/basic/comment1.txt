The only line that has to be printed.
